index.html

```diff
@@ -8338,13 +8339,58 @@ <h3>Processing actions</h3>
  input JSON, such that the actions to be performed in a single <a>tick</a>
  are grouped together.
 
+ <p>To <dfn>get parent offset</dfn> of <var>context</var>:
+ <ol class="algorithm">
+  <li>Let <var>offsetLeft</var> equal to 0 and <var>offsetTop</var> equal to 0.
+  </li>
+  <li>
+   Let <var>navigable</var> be <var>context</var>'s <a>active document</a>'s
+   [=navigable/parent=].
```

Comment:
[=navigable/parent=] is a property of a navigable and not the document. Did you mean to get the current navigable instead of a parent (e.g., https://html.spec.whatwg.org/#node-navigable)?